homepage.css

```diff
@@ -124,23 +123,75 @@ ul li:hover ul{
 	color:black;
 }
 
-.content_2{
-	background-color: white;
+
+.content_4{
 	grid-column-start: 1;
 	grid-column-end: span 4;
-	z-index: 2;	
-	height: 150px;
-	text-align:center;
+	background-color: white;
+	z-index: 2;
+	height:600px		
 }
 
+.content_4{
+	width: 100%;
+	display: grid;
+    grid-gap: 2%;
+	grid-template-columns: 5% 20% 20% 20% 20%;
+	grid-template-rows: auto auto;
+}
 
+.images1{
+	height:214px;
+	background-color:white;
+	margin-top:50px;
+}
 
-.content_2 p{
-	font-size:16px;
-	color:black;
+.images2{
+	height:214px;
+	background-color:white;
+	margin-top:50px;
 }
 
-.content_2 h1{
-	font-size:24px;
-	color:black;
+.images3{
```

Comment:
I set up a div box for the third img in the content_4 (like height, background, margin)